docs/UserGuide.md

```diff
@@ -410,79 +410,120 @@ Deletes the specified delivery from the address book.
+- Spacing between `-d` and `INDEX` is not compulsory.
+- Only one delivery can be deleted one command.
+  - `delete -d 1 3 5` is not allowed.
+
 </box>
 
 Examples:
-* `list` followed by `delete -d 2` deletes the 2nd delivery in the address book.
-* `find -d /pro bread` followed by `delete -d 1` deletes the 1st delivery in the results of the `find` command.
+- `delete -d 2`
+- `find -d pro/ bread` followed by `delete -d 1` deletes the 1st delivery in the results of the `find` command.
+
+Expected output:
+- Delivery at index 2 of the displayed list will be deleted, assuming there are at least 2 deliveries in the displayed list. Otherwise, an error message will be shown.
+- Delivery at index 1 of the displayed list will be deleted, assuming there is at least 1 delivery in the displayed list after the find command is executed. Otherwise, an error message will be shown.
 
 #### Here's how it would look like in the app:
 ![delete delivery command](images/deleteDeliveryCommand.png)
 
 ### Find a delivery: `find -d`
 
-Find deliveries based on attributes of the delivery, like the delivery date and time, delivery status, supplier and product.
+Find deliveries based on attributes of the delivery, like the delivery date and time, status, supplier and product.
 
 Format: `find -d on/DELIVERY_DATE_TIME stat/STATUS s/SUPPLIER_INDEX pro/PRODUCT`
 
+<box type="details" seamless>
+
 Parameters:
 
-- `on/DELIVERY_DATE_TIME`: Must be in dd-mm-yyyy hh:mm format and must not be blank.
-- `stat/STATUS`: Must be one of the following: PENDING, DELIVERED, CANCELLED and must not be blank.
-- `s/SUPPLIER_INDEX`: Must be a number greater than 0 and must not be blank.
-- `pro/PRODUCT`: Must be alphanumeric, can include spaces but must not start with a space, and must be between 1 and 50 (inclusive) characters long.
-<box type="tip" seamless>
+- `on/DELIVERY_DATE_TIME`: `DELIVERY_DATE_TIME` must be in dd-MM-yyyy hh:mm format and must not be blank.
+- `stat/STATUS`: `STATUS` must be one of the following: PENDING, DELIVERED, CANCELLED and must not be blank.
+- `s/SUPPLIER_INDEX`: `SUPPLIER_INDEX` must be a number between 1 and the total number of suppliers displayed (inclusive), and cannot be blank.
+- `pro/PRODUCT`: `PRODUCT` must be alphanumeric, only contain between 1 and 50 (inclusive) characters, and spaces are also allowed.
+    - One space is counted as one character.
+    - `PRODUCT` cannot be made up of only spaces.
+
+</box>
+
+<box type="warning" seamless>
 
 **Warnings**:
-- A spacing between `find` and `-d` is compulsory
-- At least one prefix and parameter must be given
-- No duplicate prefix can be used
-- Find result(s) will contain/satisfy all the given parameters
-- Parameters used are case-insensitive
+- At least one space is needed between `find` and `-d`.
+- At least one parameter must be given.
+- At least one space is needed between `-d` and the first parameter.
+- When using more than one parameter, at least one space is needed between parameters.
```

Comment:
Would it be better to say that 
At least one space is needed between each parameter and the next prefix. ?